### DISCUSSION TITLE
Is it possible to customise names of aux output switches on the GUI?

### DISCUSSION THREAD

**Discussion by axg20202** [terjeio/ioSender#271] [General]
Is there a way to change the names of Flood, Mist and Fan controls in the GUI? I have these triggering different functions, so it would be great if they could be customised somehow.

**Comment by terjeio** [comment]
No, unless you [build your own version](https://github.com/terjeio/ioSender/wiki/Building) from source.

**Reply by axg20202** [reply]
OK, thanks. Will take a look.

**Reply by phil-barrett** [reply]
You could make macros and give them appropriate names.

**Reply by terjeio** [reply]
Unlike macros the checkboxes can be used when a gcode program is running - so macros will only be a partial solution?

**Comment by axg20202** [comment]
Thanks both.  I did take a look at digging into the coding but beyond tweaking config files and compiling, it's a bit out of my skillset.

**Reply by terjeio** [reply]
In [this file](https://github.com/terjeio/ioSender/blob/master/CNC%20Controls/CNC%20Controls/CoolantControl.xaml), labels are from the `Content` tag. Issue #259 contains info on how to get compilation to work.

**Comment by axg20202** [comment]
Thanks! Worked like a charm once I'd sorted out the references.
![image](https://user-images.githubusercontent.com/58083169/218859925-efdee544-54fc-45c0-b6d4-3966af4405e6.png)
